README.md

```diff
@@ -26,18 +24,14 @@ ReadTheDocs](https://indica-ukaea.readthedocs.io/en/latest/), which is also stil
     - *pip install poetry==1.7*
     - *poetry update*
```

Comment:
I know it wasn't changed in this PR, but I only just noticed this says `poetry update`. Should be `poetry install` or `poetry install --sync`.